frankenstein/ring.go

```diff
@@ -111,3 +120,35 @@ func (r *Ring) updateSortedHashes() {
 	r.sortedHashes = hashes
 	r.circle = circle
 }
+
+// Describe implements prometheus.Collector.
```

Comment:
Oh I see; this is just a slightly different way of exposing prometheus metrics, for stuff that should be calculated on scrape (as opposed to constantly maintained).